contrib/prosody/files/prosody

```diff
@@ -0,0 +1,8 @@
+# prosody system service
+
+type = bgprocess
```

Comment:
... just make a proper process service

Reply:
it didn't work

Reply:
didn't work how

Reply:
just using `/usr/bin/prosody` as the command line should be completely enough

Reply:
that's what I started with early and again.. for some reason it just doesn't work